### PR TITLE
oopsy: don't report share for invulning tanks

### PR DESCRIPTION
**Pull request by g0shu**

### CODE REVIEW

**Comment by g0shu** [comment]
uhhh i don't entirely get what that failed test is but looks like it's not on my end?

File: ui/oopsyraidsy/data/06-ew/raid/p12s.ts [hunk 1]
```diff
@@ -93,6 +93,22 @@ const triggerSet: OopsyTriggerSet<Data> = {
...
+      id: 'P12S White Flame Share',
+      type: 'Ability',
+      netRegex: NetRegexes.ability({ id: '82F0' }),
+      condition: (data, matches) => !data.IsImmune(matches.targetId),
+      mistake: (_data, matches) => {
+        return {
+          type: 'warn',
+          blame: matches.target,
+          reportId: matches.targetId,
+          text: matches.ability,
```
Comment (quisquous) [inline ui/oopsyraidsy/data/06-ew/raid/p12s.ts:107]:
```suggestion
          text: GetShareMistakeText(matches.ability, numTargets),
```

You'll need to `import { GetShareMistakeText } from '../../../oopsy_common';` too. This is what `AddShareTriggers` does for the `shareFail` etc helpers. Maybe I need a `GetShareMistakeFunc` that hands back that whole helper function.

One question I have for you is ... what happens if this hits two invulning tanks and a hypothetical machinist who stands too close? Should it just say `(share x3)` on the machinist (but nobody else)? Or something else?

**Comment by quisquous** [comment]
> uhhh i don't entirely get what that failed test is but looks like it's not on my end?

I re-ran it. Just looks like the bot was being weird. idk

**Comment by quisquous** [comment]
Thank you!!

**Pull request merged**